projects/ngx-mat-intl-tel-input/src/lib/ngx-mat-intl-tel-input.component.ts

```diff
@@ -349,7 +350,7 @@ export class NgxMatIntlTelInputComponent extends _NgxMatIntlTelInputMixinBase
   }
 
   onContainerClick(event: MouseEvent) {
-    if ((event.target as Element).tagName.toLowerCase() !== 'input') {
+    if (!this.enableWCAGAccessibility && (event.target as Element).tagName.toLowerCase() !== 'input') {
```

Comment:
this is removing the earlier functionality of focusing the input if clicked on the element somewhere else